src/features/resourceFilters/ResourceFilterSelector.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+import type { IResourceList } from "@ahryman40k/ts-fhir-types/lib/R4";
+import {
+  Checkbox,
+  FormControlLabel,
+  FormGroup,
+  FormLabel,
+  Paper,
+} from "@mui/material";
+import { makeStyles } from "@mui/styles";
+import { useTranslation } from "react-i18next";
+
+import { useAppDispatch, useAppSelector } from "app/store";
+
+import {
+  resourceFilterSet,
+  selectResourceFilters,
+} from "./resourceFilterSlice";
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    padding: theme.spacing(3),
+  },
+}));
+
+type ResourceFilterSelectorProps = {
+  filters: Set<IResourceList["resourceType"]>;
+};
+
+const ResourceFilterSelector = ({
+  filters,
+}: ResourceFilterSelectorProps): JSX.Element => {
+  const classes = useStyles();
+  const dispatch = useAppDispatch();
+  const { t } = useTranslation();
+  const removedFilters = useAppSelector(selectResourceFilters);
+
+  const handleFilterClick = (filter: IResourceList["resourceType"]) => () => {
+    dispatch(resourceFilterSet(filter));
+  };
+
+  return (
+    <Paper className={classes.container}>
+      <FormLabel>{t("display")}</FormLabel>
+      <FormGroup>
+        {[...filters].map((filter) => (
```

Comment:
do you need to copy filters like this ? out of curiosity: why ?

Reply:
Because filters is a `Set` that ensures all items are unique. And because of this it's not possible to use `map` without converting it to array type.